docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -134,22 +95,19 @@ In this example, using the same data, with the min and max calculated for each i
 
 This is not only useful in the stat visualization; gauge, bar gauge, and status history visualizations, table cells formatted by thresholds, and gauge table cells all benefit from this addition.
 
-### Generative AI features for dashboards
+### New browse dashboards
```

Comment:
Copied across the latest changes for new Browse Dashboards https://github.com/grafana/grafana/pull/76567 👍🏻